### PR TITLE
Add myforce-angularjs-dropdown-multiselect@1.10.2 w/ git autoupdate

### LINKED ISSUE #9613: [Request] Add angularjs-dropdown-multiselect
**Library name:** myforce-angularjs-dropdown-multiselect
**Git repository url:** https://github.com/myforce/angularjs-dropdown-multiselect
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:** http://myforce.github.io/angularjs-dropdown-multiselect/
**Wanna say something? Leave message here:** This is very important with me :)

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
Pull request for issue: #9613 
@kennynaoh Please help me review this pull request, thank you.
- use git auto-update because there is only latest release on npm.
- I didn't use `https` for homepage because they are different between `http` and `https`.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/myforce/angularjs-dropdown-multiselect
 * Official website (optional, not the repository): http://myforce.github.io/angularjs-dropdown-multiselect/
 * NPM package url (optional): https://www.npmjs.com/package/angularjs-dropdown-multiselect
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 20
   - Count of watchers: 10
   - Count of forks: 318
 * NPM download stats (optional):
   - Downloads in the last day: 55
   - Downloads in the last week: 314
   - Downloads in the last month: 1434

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

w/ git autoupdate, close #9613, cc @dotansimha @pkempenaers

### CODE REVIEW

**Comment by kennynaoh** [comment]
LGTM

File: ajax/libs/myforce-angularjs-dropdown-multiselect/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "myforce-angularjs-dropdown-multiselect",
+  "filename": "angularjs-dropdown-multiselect.min.js",
+  "version": "1.10.2",
+  "author": "pkempenaers@myforce.be",
+  "description": "This directive gives you a Bootstrap Dropdown with the power of AngularJS directives.",
+  "homepage": "http://myforce.github.io/angularjs-dropdown-multiselect/#/",
```
Comment (PeterDaveHello) [inline ajax/libs/myforce-angularjs-dropdown-multiselect/package.json]: https

Comment (pvnr0082t) [inline ajax/libs/myforce-angularjs-dropdown-multiselect/package.json]: @PeterDaveHello The content are different between `http` and `https`.

Comment (PeterDaveHello) [inline ajax/libs/myforce-angularjs-dropdown-multiselect/package.json]: It's blocked by your browser.

**Comment by PeterDaveHello** [comment]
This package doesn't have enough popularity, and we should work on the old issues first.

**Pull request merged**